### ISSUE TITLE
Membersite has problems to add channel, but TA can

### ISSUE THREAD

**Issue by DunklerPhoenix** [tubearchivist/members#2]
Channel: UCZzGtvgIcthK2cOGvwGGbbA

Heho
I have here a very special channel. Because of legal reasons it is only available in Germany.
The funny thing is that my finnish server can add it and download videos, but the members website says "Channel import process finished, please refresh." without any error message. After a refresh of the page the channel isn't in the subscription list of the members site.

Is it because of this special case or doesn't the process on the members page have any error messages?

-dp

**Comment by bbilly1** [comment]
Yes, that channel is not available when you open the URL. That won't work.
